Internal/processProviders/go_mod.go

```diff
@@ -3,9 +3,52 @@ package processProviders
 const gomodcontent = `
 module terraform-provider-junos-%+v
 
-go 1.14
+go 1.17
 
 require (
         github.com/davedotdev/go-netconf v0.1.5
-        github.com/hashicorp/terraform-plugin-sdk v1.17.2
-)`
+	github.com/hashicorp/terraform-plugin-log v0.4.0
+        github.com/hashicorp/terraform-plugin-sdk/v2 v2.16
+)
+
+require (
```

Comment:
What is the License for each of these new modules that are added as dependencies?

Reply:
I have removed the Indirect dependency. User will be prompted to do `go mod tidy` to update these before doing go build.